storybook/pages/SwapInputPanelPage.qml

```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+import shared.stores.send 1.0
+
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.panels 1.0
+import AppLayouts.Wallet.controls 1.0
+
+import Models 1.0
+import Storybook 1.0
+
+import SortFilterProxyModel 0.2
+
+SplitView {
+    id: root
+
+    Logs { id: logs }
+
+    QtObject {
+        id: d
+
+        readonly property string networksChainsCurrentlySelected: {
+            // let supportedNwChains = []
+            // for (let i = 0; i< networksRepeater.count; i++) {
+            //     if (networksRepeater.itemAt(i).checked && networksRepeater.itemAt(i).visible)
+            //         supportedNwChains.push(networksRepeater.itemAt(i).chainID)
+            // }
+            // return supportedNwChains.join(":")
+            return "1:5:10:420:421613"
+        }
+
+        property WalletAssetsStore walletAssetStore: WalletAssetsStore {
+            assetsWithFilteredBalances: SubmodelProxyModel {
+                sourceModel: d.walletAssetStore.groupedAccountsAssetsModel
+                submodelRoleName: "balances"
+                delegateModel: SortFilterProxyModel {
+                    sourceModel: submodel
+                    filters: FastExpressionFilter {
+                        expression: {
+                            d.networksChainsCurrentlySelected
+                            return d.networksChainsCurrentlySelected.split(":").includes(model.chainId+"")
+                        }
+                        expectedRoles: ["chainId"]
+                    }
+                }
+            }
+        }
+
+        property TransactionStore store: TransactionStore {
+            walletAssetStore: d.walletAssetStore
+            //showCommunityAssetsInSend: false // FIXME in reality, this is in TokensStore
```

Comment:
commented out so remove?

Reply:
Well it's more of a reminder for us; I wasn't quite sure we should support showing community assets here; if not, we ought to configure the assets model in that way. Currently it's done in a slightly different way between the real `TransactionStore` and the SB/mocked one